nodejs/npm.cmd

```diff
@@ -0,0 +1,4 @@
+@echo off
+SET PATH=%~dp0;%PATH%
+SET npm_config_git=%~dp0git.cmd
+"%~dp0node" "%~dp0..\packages\Npm.3.5.2\node_modules\npm\bin\npm-cli.js" %*
```

Comment:
is it possible to inline those 3 files into the 'JS Unit-Testing' step and rewrite in powershell?
With the Fetch-Package command it will not be needed to hardcode the full path to package (as the command is returning path to fetched package)